package.json

```diff
@@ -16,7 +16,7 @@
     "api:testWatch": "cd api && npm run testWatch",
     "api:watch": "cd api && tsc --watch",
     "api:wsl": "nodemon --legacy-watch api/bin/www",
-    "client": "cd client && npm run start",
+    "client": "cd client && npm run dev",
```

Comment:
Would it make sense to keep the default client task as `start` instead of `dev` and then create a separate `client:dev` script which is called by the top-level `dev` script? It seems like this might be a little more predictable/consistent.

Reply:
Made change